libr/core/panels.c

```diff
@@ -106,6 +106,28 @@ static const char **menus_sub[] = {
 	NULL
 };
 
+static void Layout_run(RPanels *panels);
+static void Panel_print(RConsCanvas *can, RPanel *panel, int color);
```

Comment:
Why uppercase letter?

Reply:
it's how it had been before me

Reply:
but in fact, those uppercase look weird to me too so i fixed 'em now, thanks :)